### PR TITLE
test: R test with animal shelter data

### PR DESCRIPTION
**Pull request by satai**

### CODE REVIEW

**Review by honzasterba — APPROVED** [review]

**Review by michalkurka — APPROVED** [review]

**Review by michalkurka — APPROVED** [review]

File: h2o-r/tests/testdir_algos/coxph/runit_coxph_shelter.R [hunk 1]
```diff
@@ -0,0 +1,50 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../../scripts/h2o-r-test-setup.R")
+
+test.CoxPH.heart <- function() {
```
Comment (grigory93) [inline h2o-r/tests/testdir_algos/coxph/runit_coxph_shelter.R:4]: test.CoxPH.animalShelter maybe?

File: h2o-r/tests/testdir_algos/coxph/runit_coxph_shelter.R [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    expect_equal(output$wald.test, coxph.r$wald_test, tolerance = 1e-8)
+     
+    # smoke tests
+    print(extractAIC(coxph.h2o))
+    print(logLik(coxph.h2o))
+    print(vcov(coxph.h2o))
+}
+
+doTest("CoxPH: Heart Test", test.CoxPH.heart)
```
Comment (grigory93) [inline h2o-r/tests/testdir_algos/coxph/runit_coxph_shelter.R:50]: the same: change from Heart Test to Animal Shelter

**Review by grigory93 — COMMENTED** [review]
test still carries 'heart' in the name

File: h2o-r/tests/testdir_algos/coxph/runit_coxph_shelter.R (hunk 2 above)
Comment (satai) [reply h2o-r/tests/testdir_algos/coxph/runit_coxph_shelter.R:50]: You are right: https://github.com/h2oai/h2o-3/pull/5333

**Pull request merged**